lakefs_provider/hooks/lakefs_hook.py

```diff
@@ -133,3 +144,25 @@ def create_symlink_file(self, repo: str, branch: str, location: str = None)  ->
             kwargs["location"] = location
 
         return client.metadata.create_symlink_file(repository=repo, branch=branch, **kwargs)["location"]
+
+    def test_connection(self):
+        "TO do"
+        """Test  Connection"""
+        try:
+            # client = self.get_conn()
+            conn = self.get_connection(self.lakefs_conn_id)
+            import requests
+            import json
+            url = conn.host+"/api/v1/auth/login"
+            payload = json.dumps({
+                "access_key_id": conn.extra_dejson.get("access_key_id", None),
+                "secret_access_key": conn.extra_dejson.get("secret_access_key", None)
+            })
+            headers = {'Content-Type': 'application/json'}
+            response = requests.request("POST", url, headers=headers, data=payload)
+            if response.status_code == 200:
+                return True, "Connection successfully tested"
+            else:
+                return False, "Connection Failed, Unauthorized"
```

Comment:
It might not be unauthorized.  So we should return a description of the status (`import http.client`, followed by `client.responses[response.state_code]`, or alternatively the first 200 or so bytes of the response.

Reply:
I have tested it with correct and incorrect credentials. With incorrect credentials it shows  "Connection Failed, Unauthorized" under a red banner.
please let me know if this is what you asked.

Reply:
Great testing!  But there are _other_ ways to fail!  For instance, suppose I type the URL incorrectly.  I may get a 404 Not Found, or anything else really.  This code will always say "Unauthorized".  As a user of the connection on Airflow it will not tell me what to do, and will even mislead me and make me think I have some other problem.

So I want to report the most informative error here.